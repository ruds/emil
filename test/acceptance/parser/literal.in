123 ; 123i ; 123f ;
"hello" ;
R"(multiline
literal
)" ;
#"\U01f603" ; #"a" ;
