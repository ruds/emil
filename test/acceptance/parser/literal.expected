0001 (ExprTopDecl
         :expr
         (BigintLiteralExpr
             :val
             123))
0001 (ExprTopDecl
         :expr
         (IntLiteralExpr
             :val
             123i))
0001 (ExprTopDecl
         :expr
         (FpLiteralExpr
             :val
             123.000000))
0002 (ExprTopDecl
         :expr
         (StringLiteralExpr
             :val
             "68656C6C6F"))
0003 (ExprTopDecl
         :expr
         (StringLiteralExpr
             :val
             "6D756C74696C696E650A6C69746572616C0A"))
0006 (ExprTopDecl
         :expr
         (CharLiteralExpr
             :val
             #"\U01F603"))
0006 (ExprTopDecl
         :expr
         (CharLiteralExpr
             :val
             #"a"))
0007 (EndOfFileTopDecl)
